MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include pystac/py.typed pystac/html/*.jinja2
+exclude tests/*
```

Comment:
do you need to exclude benchmarks as well?

Reply:
Don't think so ... the `benchmarks/` package is avoided by the `exclude` statement in the `pyproject.toml`. We have to exclude `tests/` in `MANIFEST.in` (as well as `pyproject.toml`) because **setuptools** wants to slurp `tests/` by default: https://setuptools.pypa.io/en/latest/deprecated/distutils/sourcedist.html#specifying-the-files-to-distribute

I checked my work with the following one-liner: `python -m build && tar tf dist/*.tar.gz && unzip -l dist/*.whl`